crates/shared/src/sources/uniswap_v3/pool_fetching.rs

```diff
@@ -229,6 +245,7 @@ async fn update_recently_used_outdated_pools(
     update_size: Option<usize>,
 ) {
     while let Some(inner) = inner.upgrade() {
+        tracing::debug!("periodical update started");
```

Comment:
Nit: periodical is a term used for periodically publishing articles, might be better to use just "periodic"

cf - https://grammarist.com/usage/periodic-periodical/

But please feel free to ignore.